tests/conftest.py

```diff
@@ -86,27 +87,34 @@ def target_api_config():
 
 
 @pytest.fixture(scope='function')
```

Comment:
Lol so first, can you please put the mocking behavior back to: 

```suggestion
# top of module
# Mock response from dataservice /swagger
mock_dataservice_schema = read_json(
    os.path.join(TEST_DATA_DIR, 'mock_dataservice_schema.json')
)

@pytest.fixture(scope='function')
def auto_transform_stage(caplog):
    """
    Re-usable fixture for tests. Use this one for all tests that need
    the auto transform stage and you don't want to worry about setting it up.
    """
    # Set pytest to capture log events at level INFO or higher
    caplog.set_level(logging.DEBUG)

    # Mock get_open_api_v2_schema to always return the schema
    patcher = mock.patch(
        'kf_lib_data_ingest.common.misc.get_open_api_v2_schema',
        return_value=mock_dataservice_schema)
    patcher.start()

    yield AutoTransformStage(
        KIDS_FIRST_CONFIG,
        target_api_url=DEFAULT_TARGET_URL,
        ingest_output_dir=TEST_INGEST_OUTPUT_DIR)

    patcher.stop()
    delete_ingest_outputs(TEST_INGEST_OUTPUT_DIR)


@pytest.fixture(scope='function')
def guided_transform_stage(caplog):
    """
    Re-usable fixture for tests. Use this one for all tests that need
    the guided transform stage and you don't want to worry about setting it up.
    """
    # Set pytest to capture log events at level INFO or higher
    caplog.set_level(logging.DEBUG)

    # Mock get_open_api_v2_schema to always return the schema
    patcher = mock.patch(
        'kf_lib_data_ingest.common.misc.get_open_api_v2_schema',
        return_value=mock_dataservice_schema)
    patcher.start()

    yield GuidedTransformStage(
        TRANSFORM_MODULE_PATH, KIDS_FIRST_CONFIG,
        target_api_url=DEFAULT_TARGET_URL,
        ingest_output_dir=TEST_INGEST_OUTPUT_DIR)

    patcher.stop()
    delete_ingest_outputs(TEST_INGEST_OUTPUT_DIR)
```

Reply:
The reason why all the tests still pass without mocking behavior is because of our `request_retry_session method`, which gets called during transform stage's run (right before handle nulls). This method gracefully returns when it cannot connect to the dataservice. 

Leaving the mocking out doesn't break anything but the whole point of these two fixtures in particular is to have working transform stages that return "good" output. Then any test that needs a working transform stage (using data from test_study) can use these two fixtures.

Reply:
Btw, I could not figure out an elegant way to define both fixtures without the repetitive mocking code. I tried doing some stuff w/ the mock decorators but it wasn't working and I gave up easily.

Reply:
👍 no problem
![thecandle](https://user-images.githubusercontent.com/201996/54163064-21eee700-442e-11e9-8a09-cb98c86ea150.gif)

Reply:
LOL